apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -905,7 +950,7 @@ func (m *ModelManagerAPI) ModelInfo(ctx context.Context, args params.Entities) (
 	return results, nil
 }
 
-func (m *ModelManagerAPI) getModelInfo(ctx context.Context, tag names.ModelTag, withSecrets bool, modelCreator bool) (params.ModelInfo, error) {
+func (m *ModelManagerAPI) getModelInfo(ctx context.Context, tag names.ModelTag, withMachines, withSecrets bool, modelCreator bool) (params.ModelInfo, error) {
```

Comment:
Would be nice to see this just serve `ModelInfo` and not try to be doing two different things poorly. As per comment above.